native-code/android/index.md

```diff
@@ -77,6 +77,33 @@ For instructions on how to build and run, see
 [webrtc/examples/androidapp/README][3].
 
 
+### Using Android Studio
+
+  1. Build the project normally:
+
+     ~~~~~ bash
+     ninja -C out/Debug AppRTCMobile
```

Comment:
I thought it should be obvious but of course I can add it if you feel these instructions are unclear.